docs/reference/running/deployment/note_cloud_reset_password.rst

```diff
@@ -0,0 +1,9 @@
+.. note:: Gel Cloud: Reset the default password for the admin role
+
+    If you want to dump an existing Gel Cloud instance and restore it to a new self-managed instance, you need to change the automatically generated password for the default admin role - ``edgedb`` or ``admin``.
+    The administrator role name and its password used in the dump/restore process must be the same in both the Cloud and self-managed instances for the Gel tooling to continue functioning properly.
```

Comment:
This is a bit nit picky. Seems like the documentation should be applicable more broadly than just the cloud.

```suggestion
    The administrator role name and its password used in the dump/restore process must be the same in both the instance dumped from and the instance restored to for the Gel tooling to continue functioning properly.
```